django/db/models/utils.py

```diff
@@ -57,7 +57,7 @@ class AltersData:
     Make subclasses preserve the alters_data attribute on overridden methods.
     """
 
-    def __init_subclass__(cls, **kwargs):
+    def __init_subclass__(cls, cls_meta=None, **kwargs):
```

Comment:
This signature change is suspicious, the added param is completely ignored.

Reply:
Thanks, cls_meta needs to be absorbed here and not passed to the call to super().__init_subclass__, hence this implementation